go/cli/cmd/kagent/main.go

```diff
@@ -494,6 +654,139 @@ Example:
 		},
 	})
 
+	shell.AddCmd(&ishell.Cmd{
```

Comment:
You can remove the interactive commands, we don't plan to fully support that UX going forward